packer/vmware.json

```diff
@@ -0,0 +1,85 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "<tab> text ks=http://{{ .HTTPIP }}:{{ .HTTPPort}}/{{user `ks_file`}}.cfg<enter>"
+      ],
+      "disk_size": 10240,
```

Comment:
Unsure what the vmware.json file is for and how that's different from educationbase.json so please ignore this comment if it doesn't make sense.
The disk size here is half that of the size in educationbase.json.